src/main/java/org/apache/sysml/runtime/instructions/gpu/context/GPUContext.java

```diff
@@ -242,6 +243,10 @@ private void initializeCudaLibraryHandles() throws DMLRuntimeException {
 		if (kernels == null) {
 			kernels = new JCudaKernels();
 		}
+		// The CUDA binding methods will throw a CudaException if the CUDA function did not return cudaError.cudaSuccess.
+		JCuda.setExceptionsEnabled(true);
```

Comment:
You don't need to set `JCuda.setExceptionsEnabled` & `JCudaDriver.setExceptionsEnabled`
They are already done in `GPUContextPool`